bin/migrate

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
This "migration" is for the actual data load, not the schema "migration" as it's referred to by the tool and `/migrations` dir, right? So maybe it'd be better to call this something like "load_data" ? or at least add a comment at the top to say what it is for, and when in the pipeline it should be run?